app/src/main/java/com/futurice/freesound/network/api/ApiNetworkModule.java

```diff
@@ -69,20 +65,12 @@ static FreeSoundApi provideFreeSoundApi(@Named(API_URL_CONFIG) String url,
     @Provides
     @Singleton
     @ForFreeSoundApi
-    static Gson provideGson() {
-        return new GsonBuilder()
-                .setDateFormat(ApiConstants.DATE_FORMAT_PATTERN)
-                .registerTypeAdapterFactory(FreesoundTypeAdapterFactory.create())
-                .create();
-    }
-
-    @Provides
-    @Singleton
-    @ForFreeSoundApi
     static Moshi provideMoshi() {
         return new Moshi.Builder()
                 .add(new KotlinJsonAdapterFactory())
                 .add(new GeoLocationJsonAdapter())
+                .add(new DateAdapter())
```

Comment:
It's funny how the code from the `Rfc3339DateJsonAdapter` is originally from Jackson, then Gson, then Moshi.... and now we have our own copy.